int16 nbIterations
int8 exitFlag
float64 solvingTime
